### ISSUE TITLE
avatar_url can be undefined for a user and causes failure

### ISSUE THREAD

**Issue by Kampe** [toko-bifrost/ms-teams-deploy-card#6]
```
(node:7710) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'avatar_url' of null
    at u (/home/runner/work/_actions/edf-re/ms-teams-deploy-card/master/dist/index.js:1:356751)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(node:7710) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:7710) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by aldwyn-acn** [comment]
Hi @Kampe. I'll just replace null `avatar_url` with the official Octocat logo. Will do it tomorrow.

**Comment by aldwyn-acn** [comment]
@Kampe this has been fixed on ([v3.0.0](https://github.com/toko-bifrost/ms-teams-deploy-card/releases/tag/3.0.0)).
